apps/server/src/services/project-service/ProjectService.ts

```diff
@@ -80,14 +80,12 @@ export async function getCurrentProject(): Promise<{ filename: string; pathToFil
 
 /**
  * Private function loads a project file and handles necessary side effects
+ * @param fileName file name of the project including the extention
```

Comment:
cleanup: I am not against adding JSDocs if we feel it adds value. Here the clarification that the name includes description could be worth it. But then we should add docs for the entire interface

Also, small typo
```suggestion
 * @param fileName file name of the project including the extension
```